### PR TITLE
[FIX] Translation edit page breaks on update

### LINKED ISSUE #36: [BUG] Translation edit page breaks if all fields don't contain data.
It seems that the database will complain if certain fields are left null. This was in oversight in my earlier PR #34. A check needs to be added to allow either for an empty string to be passed or the database needs to have those fields set to default NULL. Best practice maybe to silently fail here on certain fields. I propose silently failing on:

- speaker
- episode

Current NOT NULL fields are:
- trigedasleng
- translation
- episode
- speaker
- episode

### PR DESCRIPTION
**Pull request by ZoeS17**
*FIX #36 
Make speaker and episode silently fail should their fields be left blank.

*FIX
Add translation link to the index blade

### CODE REVIEW

File: app/Http/Controllers/WebController.php [hunk 1]
```diff
@@ -104,9 +104,9 @@ public function translationEditSubmit(Request $request, $id){
             'translation' => $request->translation,
             'etymology' => $request->etymology,
             'leipzig' => $request->leipzig,
-            'episode' => $request->episode,
+            'episode' => isset($request->episode) ? $request->episode : '',
```
Comment (SKroell) [inline app/Http/Controllers/WebController.php:107]: 'episode' should default to 'other'

**Review by SKroell — CHANGES_REQUESTED** [review]
We are still redoing a ton of the database stuff, so this is a good fix for now. 

Speaker should not be marked as NOT NULL and we will address that in the database revamp. But the 'episode' column should default to 'other' to correctly display it on the translations list.

**Review by ZoeS17 — COMMENTED** [review]
Is this what you were suggesting @SKroell?

**Comment by SKroell** [comment]
> Is this what you were suggesting @SKroell?

Yes

**Review by SKroell — APPROVED** [review]

**Pull request merged**